src/services/schemas/utilities.ts

```diff
@@ -114,7 +114,7 @@ export function getSchemaPropertyPlaceholder(property: SchemaProperty): string |
     return placeholder
   }
 
-  if (isArray(placeholder)) {
+  if (isArray(placeholder) && property.meta?.component !== JsonInput) {
```

Comment:
this is kinda counter intuitive. But it makes sense

Reply:
Agreed not the finest but seems ok for now